ui/lib/apps/Statement/pages/List/index.tsx

```diff
@@ -47,8 +48,19 @@ export default function StatementsOverview() {
     allStmtTypes,
     loadingStatements,
     tableColumns,
+
+    getDownloadToken,
+    downloading,
   } = controller
 
+  async function exportCSV() {
+    const token = await getDownloadToken()
```

Comment:
I recommend this UI interact pattern:

1. Click export.
2. Show spinner / progress.
3. When successfully exported, display a Modal, with a button that links to the download link.
4. Clicking the button result in downloading the file.

This will bypass the browser popup interception when step 2 takes long time.

Reply:
Good idea, will update it.

Reply:
After trying, it is a little annoying that needs 2 steps to download, so I change another way that wouldn't trigger browser  popup interception policy (have tested it):

```
function downloadByLink(url: string) {
  const downloadLink = document.createElement('a')
  downloadLink.href = url
  downloadLink.click()
}
```

Reply:
It seems can be simplified by `window.location.href = url` as well.